pandas/core/arrays/categorical.py

```diff
@@ -523,6 +524,13 @@ def _from_inferred_categories(cls, inferred_categories, inferred_codes,
                 cats = to_datetime(inferred_categories, errors='coerce')
             elif is_timedelta64_dtype(dtype.categories):
                 cats = to_timedelta(inferred_categories, errors='coerce')
+            elif dtype.categories.is_boolean():
+                # from _true_values in parsers.pyx
+                # TODO: share declaration with parsers.pyx
```

Comment:
Cython experts, is there a way to share a declaration with a non cython file?

We have

```
cdef object _true_values = [b'True', b'TRUE', b'true']
```

changing that do a `cpdef` didn't work for some reason.

(we need this, since the PythonParser's `true_values` can apparently be none.

Reply:
you need to remove the ``cdef`` then its importable just like anything else